src/TestFramework/SafeQuery.php

```diff
@@ -33,12 +33,25 @@
 
 declare(strict_types=1);
 
-namespace Infection\Config\Guesser;
+namespace Infection\TestFramework;
 
-/**
- * @internal
- */
-interface Guesser
```

Comment:
Too bad about the ugly diff here 😢 

I've opted to drop the `Guesser` interface. It was never hinted against, and the implementations return differnt things.